src/pages/index.js

```diff
@@ -23,6 +71,10 @@ const profileEditButton = document.querySelector("#profile-edit-button");
 const profileEditModal = document.querySelector(profilePopUpSelector);
 const profileEditForm = profileEditModal.querySelector(".modal__form");
```

Comment:
**COULD BE IMPROVED**

We can get any form using their **unique** attribute `name` or `id` from `document.forms`

```js

const profileForm = document.forms["profile-form"];
const cardForm = document.forms["card-form"];

```

You need to add `name` or `id` first in `index.html` to achieve that.

More info here https://developer.mozilla.org/en-US/docs/Web/API/Document/forms